Tests/ActionTests.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ActionTests.swift
+//  Rex
+//
+//  Created by Ilya Laryionau on 4/20/16.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+@testable import Rex
+import ReactiveCocoa
+import XCTest
+import enum Result.NoError
+
+final class ActionTests: XCTestCase {
+    
+    enum TestError: ErrorType {
+        case Unknown
+    }
+
+    func testStarted() {
+        let action = Action<Void, Void, NoError> { .empty }
+
+        var started = false
+        action
+            .rex_started
+            .observeNext { started = true }
+
+        action
+            .apply()
+            .start()
+
+        XCTAssertTrue(started)
+    }
+    
+    func testCompleted() {
+        let (producer, sink) = SignalProducer<Int, TestError>.buffer(Int.max)
+        let action = Action { producer }
```

Comment:
could you change `sink` to `observer`. The `sink` nomenclature was abandoned a while back. (reactivecocoa/reactivecocoa#2475)